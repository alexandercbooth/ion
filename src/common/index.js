// @flow
export * from "./classes";
export * from "./elevation";
export * from "./intent";
