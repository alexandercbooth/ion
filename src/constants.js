// @flow
export const Elevation = {
  LOWEST: 1,
  MEDIUM: 2,
  HIGHEST: 3
};
