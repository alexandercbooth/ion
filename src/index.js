// @flow
export * from "./common";
export { Card } from "./components/card";
export { TextArea } from "./components/textArea";
