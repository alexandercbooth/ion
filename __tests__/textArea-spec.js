// @flow
import * as React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { TextArea } from "../src";

describe("TextArea", () => {
  it("renders a simple text area", () => {
    const component = shallow(<TextArea fill={true} large={true} />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
