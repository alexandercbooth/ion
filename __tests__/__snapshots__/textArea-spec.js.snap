// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea renders a simple text area 1`] = `
<div
  className="jsx-1405787395"
>
  <textarea
    className="jsx-1405787395  fill large"
  />
  <JSXStyle
    css=".fill.jsx-1405787395{display:block;width:100%;-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;}.large.jsx-1405787395{font-size:large;}"
    styleId="1405787395"
  />
</div>
`;
